sample/build.gradle

```diff
@@ -92,6 +92,14 @@ keeper {
 
   // Example: share proguard rules for L8 (library desugaring).
   enableL8RuleSharing = true
+
+  // Uncomment this line to debug the R8 from a remote instance.
+  //r8JvmArgs.addAll(Arrays.asList("-Xdebug", "-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=y"))
+
+  // Example: enable the new experimental R8 entry-point.
+  //traceReferences {
+    //arguments.set(Arrays.asList("--map-diagnostics:MissingDefinitionsDiagnostic", "error", "info"))
+  //}
```

Comment:
We use the sample as an added test form on CI as well, could we enable tracereferences via gradle property and add it to our matrix to test on CI?

Reply:
I thought about it but it's also tested in the plugin tests.
Is there any benefit to test everything twice?

Reply:
We cover some extra edge cases with the sample